simtools/schemas/model_parameters/sampled_output.schema.yml

```diff
@@ -0,0 +1,35 @@
+%YAML 1.2
+---
+title: Schema for sampled_output model parameter
+version: 0.1.0
+meta_schema: simpipe-schema
+meta_schema_url: https://raw.githubusercontent.com/gammasim/simtools/main/simtools/schemas/model_parameter_and_data_schema.metaschema.yml
+meta_schema_version: 0.1.0
+name: sampled_output
+description: |-
+  Select a specific output format (if several available). A value of 0 means to write ADC sums, a value of 1 to write ADC samples.
+data:
+  - type: int
```

Comment:
Should this be boolean?

Reply:
Yes, changed to boolean, I missed it yesterday.